src/adhocracy_frontend/adhocracy_frontend/static/i18n/core_de.json

```diff
@@ -6,6 +6,7 @@
     "TR__ACTIVATION_ERROR_SUPPORT": "Um einen neuen Link zu erhalten, kontaktiere bitte unser Support-Team und gib dabei die E-Mail-Adresse an, mit der Du Dich registrierst hast.",
     "TR__ACTIVATION_ERROR_TITLE": "Hoppla, das ging schief!",
     "TR__ACTIVATION_SUCCESS_TITLE": "Dein Account ist nun aktiv!",
+    "TR__TOGGLE_AUDIO": "Toggle Audio",
```

Comment:
> OOps fixed "Audio umschalten"

That's switching from something to something other inside the context
of audio.  "Auf Audio umschalten" would work, but then the text would
have to change to "Auf Bild umschalten" when in audio mode.

Tricky and tedious, this language.  We can also leave it as is I
guess.